openlibrary/catalog/get_ia.py

```diff
@@ -96,12 +97,12 @@ def files(identifier):
     url = item_file_url(identifier, 'files.xml')
     for i in range(5):
         try:
-            tree = etree.parse(urlopen_keep_trying(url))
+            tree = etree.fromstring(urlopen_keep_trying(url))
```

Comment:
@dherbst  @cclauss  This changes the type from an ElementTree https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.ElementTree to an Element https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.Element  -- I'm not sure of the exact implications (but the variable is named `tree`) . Has this been tested locally to confirm that nothing breaks?